src/components/ProductCategories/ProductCategories.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import AliceCarousel from "react-alice-carousel";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+const Img = styled.img`
+  width: 100%;
+`;
+
+const Title = styled.h1`
+  text-align: center;
+`;
+
+ProductCategories.propTypes = {
+  categories: PropTypes.array,
+};
+
+ProductCategories.defaultProps = {
+  categories: [],
+};
+
+function CategoryItem({ category }) {
```

Comment:
Same as before, keep one component per file, it will keep your code easy to maintain :).